### ISSUE TITLE
Support for E131

### ISSUE THREAD

**Issue by ghost** [hhatto/autopep8#300]
Continuation line unaligned for hanging indent

I thought that it's was supported before, but now it's seems to not work.

**Comment by spaceone** [comment]
Can you provide an example please?

**Comment by stepshal** [comment]
@spaceone setuptools/setup.py:90:9: E131 continuation line unaligned for hanging indent
```py
 description="Easily download, build, install, upgrade, and uninstall "
        "Python packages",
    author="Python Packaging Authority",
```
By look of this i think it's shouldn't be fixed at all.

**Comment by spaceone** [comment]
Reproducer (bash):
```
autopep8 - <<EOF
foo(
    description="Easily download, build, install, upgrade, and uninstall "
       "Python packages",
      method='asdf'
)
EOF
```

**Comment by hhatto** [comment]
fix in latest master.

It is scheduled to be released in version 1.3.
